internal/alert/provider/discord/send_test.go

```diff
@@ -0,0 +1,25 @@
+package discord
+
+import (
+	"os"
+	"testing"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/stretchr/testify/require"
+)
+
+func TestSend(t *testing.T) {
+	tk := os.Getenv("DG_TOKEN")
```

Comment:
I'm confused for use Env variables in the test. Can you refactoring this unit test without env variables?

Reply:
Yeah it's more a integration test. Sadly I found no mocks for the discord lib. Will think about that.